docs/docs/faq.mdx

```diff
@@ -1,179 +0,0 @@
----
```

Comment:
For some reason the new file is not included, generate and commit it :wink:

Reply:
uhh, lemme fix that quick